deis-cluster-coreos/azuredeploy.json

```diff
@@ -0,0 +1,361 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the of the storage account for VM OS Disk"
+            }
+        },
+        "vmNamePrefix": {

```

Comment:
This doesn't need to be a parameter. Can just hardcode. VMnames are relative to the resource group and only provide addresses.